src/main/resources/application.yml

```
@@ -0,0 +1,22 @@
+  jpa:
+    hibernate:
+      ddl-auto: validate
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+
+
+server:
+  port: 8080
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)